client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -91,6 +105,7 @@ struct ConcordClientConfig {
   TransportConfig transport;
   // BFT client descriptors
   std::vector<BftClientInfo> bft_clients;
+  std::uint16_t num_of_used_bft_clients;
```

Comment:
Isn't this `bft_clients.size()`? Why do we need an extra parameter?

Reply:
In a configuration file, we define several external clients (i.e. bft_clients) but we can actually use only some of them in the pool. This is what num_of_used_bft_clients represents and because of that, I added it as a parameter.

Reply:
Ahh, that makes sense. Thanks! With this context I can actually read it from the variable name :)